### PR TITLE
Script updates for RC1

### PR DESCRIPTION
**Pull request by kyguy**
Kafka and all of its dependencies are now provided to us in a single zip file. With this zip file, we only need to unpack its contents into ${KAFKA_HOME} directory and not worry about downloading and installing Kafka and all of its dependencies separately.

I have also updated some of the Kafka run scripts here and upstream to be able to handle jar names with versioned suffixes.

### CODE REVIEW

File: kafka/modules/kafka/2.3.0/install.sh [hunk 1]
```diff
@@ -6,9 +6,10 @@ SOURCES_DIR=/tmp/artifacts
 LICENSE_DIR=/root/licenses
 PRODUCT_LICENSE_DIR=${LICENSE_DIR}/${COM_REDHAT_COMPONENT}
 
-tar xvfz "${SOURCES_DIR}/kafka.tar.gz" -C ${KAFKA_HOME} --strip-components=1
-
-cp -r ${SOURCES_DIR}/opentracing-kafka-client.jar ${KAFKA_HOME}/libs/
+# Copy contents of zip without the root dir
+TMP=$(zipinfo -1  ${SOURCES_DIR}/kafka.zip | grep -oE '^[^/]+' | uniq)
```
Comment (ppatierno) [inline kafka/modules/kafka/2.3.0/install.sh]: we are really missing the strip-components option that tar tool has :-(

File: kafka/modules/kafka/base/scripts/kafka_connect_run.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ fi
 
 # enabling Tracing agent (initializes Jaeger tracing) as Java agent
 if [ "$STRIMZI_TRACING" = "jaeger" ]; then
-    export KAFKA_OPTS="$KAFKA_OPTS -javaagent:$(ls $KAFKA_HOME/libs/tracing-agent.jar)=jaeger"
+    export KAFKA_OPTS="$KAFKA_OPTS -javaagent:$(ls $KAFKA_HOME/libs/tracing-agent*.jar)=jaeger"
```
Comment (ppatierno) [inline kafka/modules/kafka/base/scripts/kafka_connect_run.sh]: why this change and so the relationship with same change in Strimzi upstream?

Comment (kyguy) [reply kafka/modules/kafka/base/scripts/kafka_connect_run.sh]: Not sure if the patch upstream will be accepted, and if it is if it will be cherry-picked for the next 0.14.0 RC release

File: kafka/modules/kafka/2.3.0/install.sh (hunk 1 above)
Comment (kyguy) [reply kafka/modules/kafka/2.3.0/install.sh]: Yea, unfortunately unzip doesn't have the "--strip-components=1" equivalent. This is the best we can do with unzip from what I understand. @scholzj we are tied to zips for the Kafka dependencies, correct?
